src/definitions/modules/calendar.js

```diff
@@ -789,6 +800,29 @@ $.fn.calendar = function(parameters) {
         },
 
         helper: {
+          isDisabled: function(date, mode) {
+            return mode === 'day' && (settings.disabledDaysOfWeek.includes(date.getDay()) || settings.disabledDates.some(function(d){
```

Comment:
`includes()` is not supported in IE11.
Please replace by `settings.disabledDaysOfWeek.indexOf(date.getDay())!==-1`.  I missed this in #199 where it was originally implemented.

Reply:
Woops, always forget to check this IE11 compatibility 😞